lib/core/view.coffee

```diff
@@ -119,6 +121,7 @@ module.exports = class KDView extends KDObject
     @setLazyLoader lazyLoadThreshold  if lazyLoadThreshold
     @setTooltip tooltip               if tooltip
     @setDraggable draggable           if draggable
+    @setAttribute 'draggable', yes    if droppable
```

Comment:
I wanted to use different parameter from 'draggable'. Because it may be used somewhere in framework.